src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Server Data Encoding</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QComboBox" name="comboBox_encoding">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="toolTip">
```

Comment:
Going through what I put there by paragraph:
1. The first paragraph was an attempt to explain why this control is necessary and how the various choices come about, it indirectly suggests why UTF-8 is the "one-encoding to, in the darkness, bind them" and also suggest that it is a more complex thing to handle!

2. Invites the user to see what the settings that the MUD offers and suggests they try the word "charset" as one possibility.  Having found out they will need to choose the one they want and _tell the MUD server to use it_ before setting Mudlet to match.

3. Advises them what the symptoms of a mis-match between Mudlet and the MUD server will show up as (characters getting replaced with the '�' symbol).

4. Is temporary to warn that if there are any characters that are getting consistently displayed wrongly we might have an error in one of those encoding tables - (IIRC there are one or two characters that may be subject to "regional variations" and it could be that the user is not in the right "region" for a particular table to be right for them - this is not applicable for ASCII,  UTF-8 or ISO 8859-__1__  but might be an issue for the other encodings currently).  The paragraph is also intended to emphasis that this currently only sorts out the on-screen display and that other parts such as the trigger engine have not yet been fixed up to work with non-ASCII text.

5. Is also temporary and advises against using anything other than ASCII or ISO-8859-1 - eventually it may become a recommendation to use UTF-8!